dbcog/monster_index.py

```diff
@@ -29,6 +29,7 @@
 TREE_MODIFIER_OVERRIDE_SHEET = SHEETS_PATTERN.format(1372419168)
 CONTENT_TOKEN_ALIAS_SHEET = SHEETS_PATTERN.format(1229125459)
 SERIES_OVERRIDES_SHEET = SHEETS_PATTERN.format(959933643)
+AWOKEN_SKILL_SHEET = SHEETS_PATTERN.format(1063132392)
```

Comment:
i am really sorry to say this but i think these 4 things should be in tsutils (they will never be changed, ever)

Reply:
but actually can we put these in an existing sheet? maybe the same one as dungeons? we have too many sheets

Reply:
Will do in different PR, but it is in same sheet as everything but dungeons